packages/profile-sync-controller/package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "@metamask/profile-sync-controller",
+  "version": "0.1.0",
```

Comment:
Lets not follow SemVer until V1. There might be additional changes in the near future.

(we still need to merge the controllers in)